cdc-accelerator/feature/site/__tests__/smsTemplates.test.js

```diff
@@ -24,6 +24,13 @@ describe('Sms templates test suite', () => {
             fs.existsSync.mockReturnValue(false)
             const writeFileSyncMock = jest.spyOn(fs, 'writeFileSync').mockImplementation(() => {})
 
+            fs.readdirSync.mockImplementation((dirPath) => {
```

Comment:
the readdirSync is never called on the init function. Why adding this mock on the test 'all sms templates files are generated successfully'?
Will remove it in a future PR.